### ISSUE TITLE
Why is H9-Miner-chia-og-v2.0.0-1-windows reported as Trojan:Win32/CoinMiner.N!cl malicious virus?

### ISSUE THREAD

**Issue by RedChimae** [hpool-dev/chia-miner#740]
Why is H9-Miner-chia-og-v2.0.0-1-windows reported as Trojan:Win32/CoinMiner.N!cl malicious virus?
![Why is H9-Miner-chia-og-v2 0 0-1-windows reported as malicious virus](https://github.com/h9-dev/chia-miner/assets/143989855/1dcf42da-0f71-4a56-971a-669e67ef3942)

**Comment by gabrielchiaradia** [comment]
i cannot run it, it shows virus and windows remove it automatically

**Comment by RedChimae** [comment]
> i cannot run it, it shows virus and windows remove it automatically

Most virtual currency mining programs will be identified as viruses by anti-virus software because such programs meet the typical signatures and behavior patterns of mining viruses (CoinMiner).

Solution: If you are in a Windows system environment, you can open System Settings - Update & Security - Windows Security Center - Open Windows Security Center, click Virus & threat protection in the left column - Manage settings - Add or remove exclusions, and add the h9-miner-chia-og.exe program file and its directory to the exclusion whitelist. In this way, Windows Defender will no longer trigger intervention to report viruses.

**Comment by gabrielchiaradia** [comment]
But the older version works perfectly without doing anything

**Comment by RedChimae** [comment]
> But the older version works perfectly without doing anything

Yes, I tested the two earlier versions HPool-Miner-chia-og-v1.5.4-2-windows and HPool-Miner-chia-og-v1.6.0-1-windows. Indeed, Windows Defender will not report the virus. This was released during the HPool period. The program released after the name change to H9 contains the CoinMiner virus signature code. I don't know what the official explanation is.

**Comment by kgyorev** [comment]
h9-miner-chia-console.exe don't show indication of virus, you could use it instead gui file.

**Comment by RedChimae** [comment]
> h9-miner-chia-console.exe don't show indication of virus, you could use it instead gui file.

How to use h9-miner-chia-console.exe instead of GUI for mining configuration operations? Is there any tutorial or official documentation for reference?
